test/browser/page.test.jsx

```diff
@@ -14,29 +14,57 @@ var FakeModal = React.createClass({
 });
 
 describe("page", function() {
-  var handler, page, xhr;
+  var handler, page, xhr, originalTitle;
 
-  beforeEach(function(done) {
-    xhr = sinon.useFakeXMLHttpRequest();
-    Router.run(routes.routes, '/', function(Handler) {
+  function visitPage(url, cb) {
+    Router.run(routes.routes, url, function(Handler) {
       handler = TestUtils.renderIntoDocument(<Handler/>);
       page = TestUtils.findAllInRenderedTree(handler, function(c) {
```

Comment:
Ah, got it!  :sunglasses:  

Hopefully I will be able to find some useful methods to use as I'm getting familiar with the test framework and API!